infra/gcp/terraform/k8s-infra-oci-proxy-prod/terraform.tfvars

```diff
@@ -22,263 +22,251 @@ cloud_run_config = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-east1-docker.pkg.dev"
       },
       {
         name  = "UPSTREAM_REGISTRY_PATH",
-        value = "k8s-artifacts-prod"
+        value = "k8s-artifacts-prod/images"
       }
     ]
   }
   asia-northeast1 = {
     environment_variables = [
       {
         name  = "UPSTREAM_REGISTRY_ENDPOINT",
-        value = "https://asia.gcr.io"
+        value = "https://asia-northeast1-docker.pkg.dev"
```

Comment:
```
$ crane pull asia-northeast1-docker.pkg.dev/k8s-artifacts-prod/images/pause:3.2 /dev/null && echo $?
0
```
(AKA, this region does exist and is functioning, xref: https://github.com/kubernetes/registry.k8s.io/issues/116)